src/com/oltpbenchmark/util/PlacementBlock.java

```diff
@@ -0,0 +1,40 @@
+package com.oltpbenchmark.util;
+
+/*
+ * For GeoPartitioned clusters, the configuration specifying how the
+ * tables should be placed is a list of cloud.region.zone.min_number_of_replicas
+ * blocks. 'PlacementBlock' class represents one such c.r.z.m block.
+ * 
+ * This class is used with Gson to generate a JSON to use for tablespace
+ * creation. The field names should not be changed as they should
+ * correspond exactly with the json keys expected by the SQL syntax.
+ */
```

Comment:
Comment about variable naming is not necessary. It is possible to use cameCase naming for variables and specify alternative naming for gson serialization via `@SerializedName` annotation

Reply:
Oh I should have looked for this, totally my bad. Fixed it now, thanks a lot Dmitry!